tests/helpers/__snapshots__/test_form_helper/test_default_label_size.html

```diff
@@ -0,0 +1,11 @@
+<form method="post">
+    <div id="div_id_name" class="tbxforms-form-group">
+        <label for="id_name" class="tbxforms-label tbxforms-label--m">Name</label>
+        <span id="id_name_hint" class="tbxforms-hint">Help text</span>
+        <input type="text"
+               name="name"
+               aria-describedby="id_name_hint"
+               class="tbxforms-input tbxforms-input--text"
+               id="id_name">
```

Comment:
**Question:** Why have other `<input>`s been updated so they close with `/>`, but this one hasn't?

Reply:
It seems that the form for this test is not using `baseinput.html` for rendering - in fact, none of them are. If I change `baseinput.html` to have erroneous data attributes, the snapshot tests do not failed. The other test forms do appear to be using their respective custom templates (`checkboxes.html`, etc).

I tried this with the test forms extending `tbxforms.forms.BaseForm` as well - no difference.

Is it expected crispy forms behaviour that a plain `CharField` uses `baseinput.html` for rendering? I think we should open a new issue for this if so.

Reply:
There's an open issue for this: https://github.com/torchbox/tbxforms/issues/44 - initially identified as a comment in an open issue from 2022, though created as it's own issue in April 2023.